packages/glimmer-runtime/lib/dom/change-lists.ts

```diff
@@ -126,6 +134,27 @@ export const InputValuePropertyChangeList = {
   }
 };
 
+export const TextAreaValueChangeList = {
+  setAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
+    let textArea = <HTMLTextAreaElement>element;
+    let normalizedValue = normalizeTextValue(value);
+
+    if(textArea.childNodes.length === 0) {
+      let textNode = dom.createTextNode(normalizedValue);
+      textArea.appendChild(textNode);
+    } else {
+      let currentValue = textArea.value;
```

Comment:
I just checked, with your change initial render would be:

```html
<textarea>why would someone put text here too?hello</textarea>
```

then on updating `someValue` to "goodbye":

```html
<textarea>goodbye</textarea>
```

It seems to be most consistent to ensure that the property value is used. The PR ensures that initial render would be:

```html
<textarea>hello</textarea>
```

and update would be:

```html
<textarea>goodbye</textarea>
```